api/graylog.go

```diff
@@ -255,12 +278,15 @@ func GetTlsConfig(ctx *context.Ctx) *tls.Config {
 	return tlsConfig
 }
 
-func NewStatusRequest() graylog.StatusRequest {
+func NewStatusRequest(serverVersion *GraylogVersion) graylog.StatusRequest {
 	statusRequest := graylog.StatusRequest{Backends: make([]graylog.StatusRequestBackend, 0)}
 	combinedStatus := backends.StatusUnknown
 	runningCount, stoppedCount, errorCount := 0, 0, 0
 
 	for id, runner := range daemon.Daemon.Runner {
+		if !serverVersion.SupportsMultipleBackends() {
+			id = strings.Split(id, "-")[0]
+		}
```

Comment:
If possible, I think it would be nice not to leak the "combined" IDs to the server. Maybe we can just add an additional `configuration_id` field to the status request if the detected server version supports this.

Reply:
makes sense :+1: